.gitignore

```diff
@@ -1 +1,11 @@
 build/
+.project
+/WebRoot/js/zimbra/
```

Comment:
I have added js folders that are currently copied from zm-ajax into .gitignore as lot of untracked files were showing up while locally committing the changes .
Running ant clean explicitly seems overdue to me each time you plan to do  commit.
The gitignore file that is in the universal UI branch has same content.
https://github.com/Zimbra/zm-web-client/blob/feature/universal-ui/.gitignore

However, as discussed offline, as there is again some restructuring of repo that is going to happen for ajax client near, removed the copied zm-ajax  folders from gitignore.